rst2pdf/rson.py

```diff
@@ -88,7 +88,7 @@ class Tokenizer(list):
     # RSON syntax delimiters are tokenized separately from everything else.
     delimiterset = set(' { } [ ] : = , '.split())
 
-    re_delimiterset = ''.join(delimiterset).replace(']', r'\]')
+    re_delimiterset = ''.join(delimiterset).replace(']', r'\]').replace('[', r'\[')
```

Comment:
A "less magic" version of this would be something like:
```python3
delimiterset = set(' { } [ ] : = , '.split())
delimiter_pattern = '[{}]'.format(''.join(re.escape(c) for c in delimiterset))
```
You end up with:
```
>>> delimiter_pattern
'[:\\]\\[\\{\\},=]'
```

Reply:
(of course, this only works if no-one is using `re_delimiterset`)

Reply:
I'm not Python enough to understand the "of course" remark here. Is there a danger this change could break something for someone? I'd be happy to adopt the "less magic" version if "less magic" means "more recommended"

Reply:
I guess he simply means if no-one is relying on the variable being present. What's suggested there is indeed better and would be a great follow up. What you have is perfectly suitable for now though.

Reply:
@stephenfin Yep, that's what I meant. Sorry for the slow reply.